### PR TITLE
feat: purify urls with inline query support

### PR DESCRIPTION
**Pull request by cyunrei**
Add a new interaction method with the bot for inline queries, which adheres to the rules below:
```
[command] [argument]
```
In this PR, I have implemented a new method for purifying URLs. It also follows the rules mentioned above. 

You can simply use `@your_bot_name purify [text containing links]` to obtain purified links. This method also supports multiple links.

Unlike the previous command-based method where users send links and the bot replies with the purified links, causing both the original and purified links to appear in the chat history, the use of inline queries prevents the duplication of the same origin links in the chat history.

Attached screenshots for reference:

image

image

This PR is just the beginning for inline queries. I believe that by using this method, we can create other interesting commands.

### CODE REVIEW

File: boot/boot.go [hunk 1]
```diff
@@ -17,9 +18,12 @@ func ServePool() {
 	u.Timeout = 60
 
 	commands.Bot = bot
+	inline.Bot = bot
+
 	bot.Request(&tgbotapi.DeleteWebhookConfig{})
 	for update := range bot.GetUpdatesChan(u) {
 		commands.Handle(&update)
+		inline.Handle(&update)
```
Comment (sxyazi) [inline boot/boot.go:26]: Is it possible to integrate `inline` into `commands`?

**Comment by cyunrei** [comment]
Using aliases is a more user-friendly choice.

image

**Comment by sxyazi** [comment]
Maybe we can remove "pu"/"purify" to make it simpler and easier to use - trigger an inline query to the user when a purifiable URL is detected in the user input.

**Comment by cyunrei** [comment]
The removal of "pu"/"purify" is indeed more convenient, but it deviates from the specific requirements of the inline query format. Assuming that adding a new feature requires handling input containing a purifiable URL in a different way, the use of an inline query might introduce ambiguity to user input behavior. Therefore, the inline query should adhere to the format I previously defined: `[command] [argument]`.

**Comment by sxyazi** [comment]
I think this is not a problem as we don't need to implement variants of multiple purification features. If necessary, we can provide multiple options for users to choose from, although such cases are rare.

**Comment by cyunrei** [comment]
I think I didn't express my meaning clearly. Purify urls by inline query is just one of the functions of inline query. I want to use inline query as a function similar to shell interaction to implement puirfy urls. It is not exclusive to the entire function of inline query (input argument directly without setting the command form). For example, we can implement a function to download the video thumbnails (the purifiable URL can also be passed in as an argument). Or you can enter specific instructions to generate Sticker, etc. The above are all prepared for some future functions, so I think that directly entering the purifiable URL will make the inline query input ambiguous. This may seem redundant at the moment, what do you think of this idea?

**Comment by sxyazi** [comment]
Hi, I made some changes, would you like to give it a review?

**Comment by cyunrei** [comment]
I found no issues, but I noticed that the function name `InlineQueryResponse` in common.go should be `RespondInlineQuery`. I didn't realize it should be in the **Verb+Noun** format before. Let me fix it.

**Comment by cyunrei** [comment]
This change should be fine.

**Comment by sxyazi** [comment]
LGTM, thank you! Let's merge then.

**Pull request merged**